salt/modules/kapacitor.py

```diff
@@ -0,0 +1,147 @@
+# -*- coding: utf-8 -*-
+'''
+Kapacitor execution module.
+
```

Comment:
As long as a correct API endpoint (host + port) is configured in the minion config, nothing needs to be installed on the Salt minion - not even Kapacitor itself. Do we still need the `depends` block? If so, maybe be a bit more specific so people don't go looking for a python module named kapacitor?